chia/cmds/farm_funcs.py

```diff
@@ -239,7 +243,10 @@ def process_harvesters(harvester_peers_in: dict):
             print(f"Local Harvester{'s' if len(harvesters_local) > 1 else ''}")
             process_harvesters(harvesters_local)
         for harvester_ip, harvester_peers in harvesters_remote.items():
-            print(f"Remote Harvester{'s' if len(harvester_peers) > 1 else ''} for IP: {harvester_ip}")
+            if harvesters_hostnames[harvester_ip]:
+                print(f"Remote Harvester{'s' if len(harvester_peers) > 1 else ''} for host: {harvesters_hostnames[harvester_ip]} ({harvester_ip})")
```

Comment:
This line is too long, run `black` to make CI happy.

Reply:
Should we hide the IP when the hostname is found?

Reply:
Hmm.. yeah let's hide it rather then.